src/julia-parser.scm

```diff
@@ -1684,216 +1684,216 @@
-		   (if (eqv? c #\')
-		       #t
-		       (begin (write-char (not-eof-1 c) b)
-			      (if (eqv? c #\\)
-				  (write-char
-				   (not-eof-1 (read-char (ts:port s))) b))
-			      (loop (read-char (ts:port s))))))
-		 (let ((str (unescape-string (io.tostring! b))))
-		   (if (= (length str) 1)
-		       ;; one byte, e.g. '\xff'. maybe not valid UTF-8, but we
-		       ;; want to use the raw value as a codepoint in this case.
-		       (wchar (aref str 0))
-		       (if (or (not (= (string-length str) 1))
-			       (not (string.isutf8 str)))
-			   (error "invalid character literal")
-			   (string.char str 0)))))))))
-
-	  ;; symbol/expression quote
-	  ((eq? t ':)
-	   (take-token s)
-	   (let ((nxt (peek-token s)))
-	     (if (and (closing-token? nxt)
-		      (or (not (symbol? nxt))
-			  (ts:space? s)))
-		 ':
-		 (list 'quote (parse-atom- s)))))
-
-	  ;; misplaced =
-	  ((eq? t '=) (error "unexpected \"=\""))
-
-	  ;; identifier
-	  ((symbol? t) (take-token s))
-
-	  ;; parens or tuple
-	  ((eqv? t #\( )
-	   (take-token s)
-	   (with-normal-ops
-	   (with-whitespace-newline
-	   (cond
-	    ((eqv? (require-token s) #\) )
-	     ;; empty tuple ()
-	     (begin (take-token s) '(tuple)))
-	    ((syntactic-op? (peek-token s))
-	     ;; allow (=) etc.
-	     (let ((tok (take-token s)))
-	       (if (not (eqv? (require-token s) #\) ))
-		   (error (string "invalid identifier name \"" tok "\""))
-		   (take-token s))
-	       tok))
-	    (else
-	     ;; here we parse the first subexpression separately, so
-	     ;; we can look for a comma to see if it's a tuple.
-	     ;; this lets us distinguish (x) from (x,)
-	     (let* ((ex (parse-eq* s))
-		    (t  (require-token s)))
-	       (cond ((eqv? t #\) )
-		      (take-token s)
-		      (if (and (pair? ex) (eq? (car ex) '...))
-			  ;; (ex...)
-			  `(tuple ,ex)
-			  ;; value in parentheses (x)
-			  ex))
-		     ((eqv? t #\, )
-		      ;; tuple (x,) (x,y) (x...) etc.
-		      (parse-tuple s ex))
-		     ((eqv? t #\;)
-		      ;; parenthesized block (a;b;c)
-		      (take-token s)
-		      (if (eqv? (require-token s) #\))
-			  ;; (ex;)
-			  (begin (take-token s) `(block ,ex))
-			  (let* ((blk (parse-stmts-within-expr s))
-				 (tok (require-token s)))
-			    (if (eqv? tok #\,)
-				(error "unexpected comma in statement block"))
-			    (if (not (eqv? tok #\)))
-				(error "missing separator in statement block"))
-			    (take-token s)
-			    `(block ,ex ,blk))))
-		     #;((eqv? t #\newline)
-		        (error "unexpected line break in tuple"))
-		     ((memv t '(#\] #\}))
-		      (error (string "unexpected \"" t "\" in tuple")))
-		     (else
-		      (error "missing separator in tuple")))))))))
-
-	  ;; cell expression
-	  ((eqv? t #\{ )
-	   (take-token s)
-	   (if (eqv? (require-token s) #\})
-	       (begin (syntax-deprecation-warning s "{}" "[]")
-		      (take-token s)
-		      '(cell1d))
-	       (let ((vex (parse-cat s #\} #t)))
+          ;; char literal
```

Comment:
i thought there used to a button to toggle it? it's interesting that the link still works. i made the commit assuming it would be easy enough to turn on ignore-whitespace